mne/connectivity/spectral.py

```diff
@@ -141,7 +141,7 @@ def compute_con(self, con_idx, n_epochs):
         self.con_scores[con_idx] = plv
 
 
-class _ciPLVEst(_EpochMeanConEstBase):
+class _ciPLVEst(_EpochMeanConEstBase):  # noqa: N801
```

Comment:
whereas this method makes sense to call `ciPLVest` because it's how it's named in the paper (I assume), so we just add a `noqa` to it